js/blocks/components/tiny-mce.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * WordPress dependencies
+ */
+const { Component } = wp.element;
+const { __ } = wp.i18n;
+const { F10 } = wp.keycodes;
+
+/**
+ * Forked from the Core Classic Editor's Edit component.
```

Comment:
What do we get by forking instead of just including?

Reply:
Yeah, it'd be good to include it somehow.

But I don't think there's a good way to do that.

For example, the Classic block's `index.js` file [imports it](https://github.com/WordPress/gutenberg/blob/416c9bc9eaef6e6bceea923b6f36642746c01aba/packages/block-library/src/classic/index.js#L9):


```js
import edit from './edit';
```